common/opinion_modifiers/wc_crime_and_prison_opinions.txt

```diff
@@ -0,0 +1,27 @@
+﻿stasis_traped_me_opinion = {
+	opinion = -30
+	years = 20
+	decaying = yes
+	imprisonment_reason = yes
+}
+
+stasis_traped_close_family_opinion = {
```

Comment:
Has no localization.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117568732-25945c00-b0d3-11eb-83e6-a96c5b997db7.png)

</details>